bundles/org.openhab.binding.robonect/src/main/java/org/openhab/binding/robonect/internal/handler/RobonectHandler.java

```diff
@@ -329,6 +345,16 @@ public void initialize() {
         RobonectConfig robonectConfig = getConfigAs(RobonectConfig.class);
         RobonectEndpoint endpoint = new RobonectEndpoint(robonectConfig.getHost(), robonectConfig.getUser(),
                 robonectConfig.getPassword());
+
+        String timeZoneString = robonectConfig.getTimezone();
+        try {
+            timeZone = ZoneId.of(timeZoneString);
```

Comment:
This will throw an NPE if `timeZoneString` is null.

Reply:
Thanks, good catch!

I have fixed it.